angular.json

```diff
@@ -54,17 +55,19 @@
               "sourceMap": false,
               "extractCss": true,
               "namedChunks": false,
-              "aot": true,
+              "aot": false,
```

Comment:
we need to turn off `aot` compilation for this app in production mode. For some reason when using `aot`, the manage tasks view won't work and there is an error print out in the console. Something to do with this: https://github.com/angular/angular/issues/23878